### ISSUE TITLE
Script hangs on clover

### ISSUE THREAD

**Issue by dgk4201** [AlexanderWillner/runMacOSinVirtualBox#10]
Can't get by this screen:
![image](https://user-images.githubusercontent.com/5391507/43719916-160fdf38-995d-11e8-968c-ee535d4758d5.png)

**Comment by AlexanderWillner** [comment]
Basically, I could imaging two issues.
First, the VM does not have other drives/images mounted/configured.
Second, the APFS drive has not been integrated somehow in the clover boot image.

**Comment by dgk4201** [comment]
Here it is. Thanks for taking a look.

[VBox.log](https://github.com/AlexanderWillner/runMacOSinVirtualBox/files/2263546/VBox.log)

**Comment by AlexanderWillner** [comment]
Thanks. I meant `runMojaveVirtualbox.log`, not `VBox.log`

**Comment by AlexanderWillner** [comment]
See ```~/Library/Logs/runMojaveVirtualbox.log``` (should also be accessible via ```Console.app```)

**Comment by dgk4201** [comment]
Here it is:
[runMojaveVirtualbox.log](https://github.com/AlexanderWillner/runMacOSinVirtualBox/files/2263656/runMojaveVirtualbox.log)

Thanks.

**Comment by AlexanderWillner** [comment]
Question: are you trying to install (intentionally or not) macOS High Sierra instead of macOS Mojave?

**Comment by dgk4201** [comment]
No, but I did install Linux and windows on vb before I ran your script. I
have another Mac. I'll give it a try there.

**Comment by AlexanderWillner** [comment]
The point is, that an installer with version `12.6.03` is being used that can be found at `/Applications/Install macOS*` - I'd need to extend the logging, but my assumption is that you've a Sierra installer there.

**Comment by dgk4201** [comment]
There was an old install file in applications. I deleted it, re-downloaded
the beta from Apple, then I:

   - Deleted all existing VMs
   - Deleted VirtualBox
   - Reinstalled VirtualBox
   - Reinstalled VB extension
   - Re-ran your script again and got the same results, but I saw where
   there were files in /tmp that were already there. I deleted them and re-ran
   again. Success! But now I'm stuck in a loop: install, disk utility (erase
   and create mojave disk), reboot, disk utility comes up again. Can't seem to
   get out of it.

Regards,

Dan Kalagher
www.ixploreYellowstone.com

978-337-4201
